CHANGELOG.md

```diff
@@ -5,11 +5,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [2.93.0-beta] - 2020-03-27
```

Comment:
Do you think we should keep beta releases in the CHANGELOG?

Reply:
I think we should remove it :snowboarder: 
Will remove just now